src/pages/Detail/Detail.js

```diff
@@ -1,10 +1,255 @@
 import React, { Component } from 'react';
+import './Detail.scss';
 
 export class Detail extends Component {
+  constructor() {
+    super();
+    this.state = {
+      productInfo: [],
+      buttonValue: true,
+      buttonDropDown: true,
+      secondDropDown: true,
+      productName: '제품 이름',
+      productPrice: 0,
+      productColor: '색상 이름',
+      quantityCalculation: 1,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http:/data/data.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productInfo: data[0],
+        });
+      });
+  }
+
+  shippingButtonClick = event => {
+    const { buttonValue } = this.state;
+    this.setState({
+      buttonValue: buttonValue === true ? false : true,
```

Comment:
```suggestion
      buttonValue: !buttonValue
```
비교연산자의 값이 어차피 true or false이기에 === true등은 실행할 필요가 없습니다.
그리고 삼항연산자는 condition ? true일경우 값 : false일경우 값 형태로 이루어져 있는데 만약 불린값을 원하는 거면 굳이 삼항연산자 쓸 필요가 없습니다!!

Reply:
다른 부분도 비슷한 내용 있어보이는데 전체적으로 점검 후 수정해주세요!